src/toPairs.ts

```diff
@@ -16,22 +14,10 @@ export function toPairs<T>(object: Record<string, T>): Array<[string, T]> {
   return Object.entries(object);
 }
 
-// Inspired and largely copied from `sindresorhus/ts-extras`:
-// @see https://github.com/sindresorhus/ts-extras/blob/44f57392c5f027268330771996c4fdf9260b22d6/source/object-entries.ts
-type ObjectValues<T extends Record<PropertyKey, unknown>> =
-  Required<T>[ObjectKeys<T>];
-type ObjectEntry<T extends Record<PropertyKey, unknown>> = [
-  ObjectKeys<T>,
-  ObjectValues<T>,
-];
-type ObjectEntries<T extends Record<PropertyKey, unknown>> = Array<
-  ObjectEntry<T>
->;
-
 export namespace toPairs {
   export function strict<T extends Record<PropertyKey, unknown>>(
     object: T
-  ): ObjectEntries<T> {
+  ): Array<NonNullable<{ [K in keyof T]: [K, T[K]] }[keyof T]>> {
```

Comment:
What's the purpose of the NonNullable here? It doesn't seem like the tuple would ever be nullable.